<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EntryDialog</class>
 <widget class="QDialog" name="EntryDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>849</width>
    <height>589</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Entry Dialog</string>
  </property>
  <layout class="QFormLayout" name="form_layout_">
   <item row="0" column="0" colspan="2">
    <layout class="QHBoxLayout" name="extra_args_box_">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item>
      <widget class="QLabel" name="extra_arguments_label_">
       <property name="text">
        <string>Extra arguments</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="extra_arguments_"/>
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="2">
    <layout class="QHBoxLayout" name="db_box_">
     <item>
      <widget class="QLabel" name="compilation_database_label_">
       <property name="text">
        <string>Compilation database</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="compilation_database_path_"/>
     </item>
     <item>
      <widget class="QPushButton" name="browse_compilation_database_button_">
       <property name="text">
        <string>Browse...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <spacer name="vertical_args_entry_spacer_">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>4</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="entry_file_label_">
     <property name="text">
      <string>Entry file</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QListView" name="entry_file_list_"/>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="entry_point_label_">
     <property name="text">
      <string>Entry point</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="QListView" name="entry_point_list_"/>
   </item>
   <item row="7" column="0" colspan="2">
    <layout class="QHBoxLayout" name="outer_button_box_">
     <property name="sizeConstraint">
      <enum>QLayout::SetFixedSize</enum>
     </property>
     <item>
      <spacer name="horizontal_button_box_spacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>640</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="button_box_">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
       <property name="centerButtons">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>button_box_</sender>
   <signal>accepted()</signal>
   <receiver>EntryDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>836</x>
     <y>575</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_box_</sender>
   <signal>rejected()</signal>
   <receiver>EntryDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>836</x>
     <y>575</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
